### ISSUE TITLE
find relational subdocument data with no FeathersVuex from 2nd time on 1.7.0

### ISSUE THREAD

**Issue by flight9** [feathersjs-ecosystem/feathers-vuex#398]
### Steps to reproduce

- On feathers server side, make two models:
```
//1. As parent
module.exports = function (app) {
  const mongooseClient = app.get('mongooseClient');
  const { Schema } = mongooseClient;
  const dtChecks = new Schema({
    text: { type: String, required: true }
  }, {
    timestamps: true
  });

  return mongooseClient.model('dtChecks', dtChecks);
};
//2. As children
module.exports = function (app) {
  const mongooseClient = app.get('mongooseClient');
  const { Schema } = mongooseClient;
  const dtCheckRecords = new Schema({
    text: {
      type: String,
      required: true,
    },
    dtCheckId: {
      'type': Schema.Types.ObjectId,
      'ref': 'dtChecks',
      'required': true
    },
  }, {
    timestamps: true
  });

  return mongooseClient.model('dtCheckRecords', dtCheckRecords);
};
```
- On feathers server side, make a schema to populate dt-check-records data in dt-checks.hooks.js
```
const dtCheckRecordsSchema = {
  include: {
    service: 'dt-check-records',
    nameAs: 'records',
    parentField: '_id',
    childField: 'dtCheckId',
    asArray: true,
    query: {
      $limit: 3,
      $sort: { createdAt: -1 },
    },
  }
};
//...
  after: {
    all: [
      populate({ schema: dtCheckRecordsSchema }),
    ],
    find: [],
    get: [],
    create: [],
    update: [],
    patch: [],
    remove: []
  },
```
- On feathres vuex client side, declare relation like what examples do in store/index.js
```
export default new Vuex.Store({
  plugins: [
    service('dt-checks', {
      namespace: 'dtChecks',
      instanceDefaults: {
        records: 'DtCheckRecord'
      }
    }),
    service('dt-check-records', {
      namespace: 'dtCheckRecords',
      instanceDefaults: {
      }
    }),

    // Setup the auth plugin.
    auth({ userService: 'users' })
  ]
})
```
- On feathres vuex client side, map findDtChecks function and create a button to call testFindDtChecks()  in a vue file:
```
methods: {
    ...mapActions('dtChecks', { findDtChecks: 'find' }),
    async testFindDtChecks () {
      const result = await this.findDtChecks({ query: {} })
      console.info({ findDtChecks: result })
    }
  },
```

### Expected behavior
Every time, the subdocuments of result are with FeathersVuex

### Actual behavior
The 1st time, the subdocuments of result are with FeathersVuex(see pic):
![with-feathersvuexmodel-1130_1](https://user-images.githubusercontent.com/1070457/71392151-12016600-2642-11ea-98df-3eba5b39ff3b.png)

But from the 2nd time, the subdocuments of result are without FeathersVuex(see pic):
![without-feathersvuexmodel-1130_2](https://user-images.githubusercontent.com/1070457/71392155-19c10a80-2642-11ea-81a1-08dab80ec019.png)

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working): 1.7.0 - 1.7.2
```
    "@feathersjs/authentication-client": "^1.0.2",
    "@feathersjs/commons": "4.0.0",
    "@feathersjs/feathers": "^3.1.7",
    "@feathersjs/socketio-client": "^1.1.0",
    "feathers-vuex": "^1.7.2",
    "socket.io-client": "^2.1.1"
```
- sorry, I can't upgrade to feathers-vuex2.x or 3.x, for it's company's project, that upgrade may affect many people and their code

**NodeJS version**: 8.9.3

**Client Version**: quasar v.0.17.20

**Comment by marshallswain** [comment]
I'm sorry that you can't upgrade to 2.0.  That really puts you in a tough situation because this is definitely fixed in 2.0 and now 3.0.  I'll at least give you some guidance.  The magic to fix this bug all lies in the new `mergeWithAccessors` utility that came with 2.0+.  You can find it here: https://github.com/feathersjs-ecosystem/feathers-vuex/blob/master/src/utils.ts#L409

If you make a pull request that makes sure that all database responses that modify store data go through that function, it should maintain the Vue reactivity of the model instances instead of overwriting the records with plain objects.

**Comment by marshallswain** [comment]
I'm going to go ahead and close this, since I cannot get to it any time soon.  Please reopen if you plan on working on it.
